wordpress-mobile/WordPress-iOS
# Fix media item details layout on iPad

It's a production issue, hence I'm targeting `trunk`.

To test:

For all media types: images, video, documents, and audio

- Verify that the preview for are displayed using reasonable size on all devices and device orientations
- Verify that the previews are loaded quickly (now use `.medium` thumbnail size)
- Verify that when you tap on a preview, it displays the selected media item fullscreen

## Regression Notes
1. Potential unintended areas of impact: Site Media Details
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
